tICA/scripts/ComputeTICAFeatures.m

```diff
@@ -0,0 +1,646 @@
+function [features, other_features]=ComputeTICAFeatures(StudyFolder, GroupAverageName, SubjListName, ...
+                                                        fMRIListName, OutputfMRIName, tICAdim, ...
+                                                        fMRIProcString, tICAFeaturesProcString, ...
+                                                        Resolution, RegString, LowResMesh, ...
+                                                        ToSave,...
+                                                        hp, MRFixConcatName, RecleanMode, ...
+                                                        CorticalParcellationFile, ParcelReorderFile, NiftiTemplateFile, VascularTerritoryFile, VesselProbMapFile, MultiBandKspaceMapFile)
```

Comment:
Last set of arguments is extra long, and doesn't align with the preceding lines.